resources/js/contest-voting/entry.coffee

```diff
@@ -90,6 +89,19 @@ export class Entry extends React.Component
             target: '_blank'
             i className: 'fas fa-fw fa-lg fa-external-link-alt'
 
+  renderTitle: (entry_title) ->
```

Comment:
entry title should just be set inside this function and make sure to not put `a` inside `a`.

(also new variable names should be camelCase)